### ISSUE TITLE
Splitting RGB from Alpha - different compositing algorithms.

### ISSUE THREAD

**Issue by dawidd4** [chaiNNer-org/chaiNNer#1773]
**Motivation**
I played with upscalers a little, those trained by the AI, and I noticed, the free upscalers don't handle the alpha channel quiet well. So I split the RGB and A channels, scaled them up by the trained upscale models, and I wanted to merge them in GIMP after making some manual improvements there.
The end result was that merging the channels together, the middle tones of transparency became even more transparent. (I didn't know where the issue was, I've found the problem by chatting with GPT).

**Description**
I think of an option in the settings menu to choose the so called "compositing algorithm" for seperating the RGB and alpha channels.

**Alternatives**
I'm now aware of this issue (read: property) of splitting the RGBA channels in one program and merging them in another. From now on I will merge them back in ChaiNNer before further image editing in GIMP.
(Should ChaiNNer use a different compositing algorithm maybe, the same one as in GIMP?)

**Comment by joeyballentine** [comment]
You can easily accomplish this in chainner by just splitting the transparency, upscaling, and merging back together, which is why we don't just have an option for it. It's super easy to do manually

**Comment by dawidd4** [comment]
After applying a linear gradient in the mask layer, from left to right, from black to white, where 127 and 128 are in the middle of this gradient. on an image that contains a black and a white horizontal stripe, separatly in ChaiNNer and GIMP,
I come to the following conclusion:
The 50% transparency is exact in the middle of these stripes when merging all channels in ChaiNNer, while in GIMP the 50% transparency is somewhere at 75% on the right of these stripes.
That means, the transparency is evenly overlaid in ChaiNNer, but not in GIMP.
TL;DR - **ChaiNNer wins**

**Comment by RunDevelopment** [comment]
I'm going to close this issue now, as the issue seems to have been resolved.
